File: src/main/java/com/example/demo/model/Employee.java
package com.example.demo.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@Table
@Data
@Entity
@NoArgsConstructor
public class Employee {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String email;
    private String name;
    private int phone;
    private String departement;

    public int getId() {
        return id;
    }

    public String getEmail() {
        return email;
    }

    public String getName() {
        return name;
    }

    public int getPhone() {
        return phone;
    }

    public String getDepartement() {
        return departement;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setPhone(int phone) {
        this.phone = phone;
    }

    public void setDepartement(String departement) {
        this.departement = departement;
    }
}

File: angularui/src/app/services/api.services/models/employee.ts
export class Employee {
  id?: number;
  email?: string;
  name?: string;
  phone?: number;
  departement?: string;

  constructor(id?: number, email?: string, name?: string, phone?: number, departement?: string) {
    this.id = id;
    this.email = email;
    this.name = name;
    this.phone = phone;
    this.departement = departement;
  }
}
